docs/officers/payments/accept.rst

```diff
@@ -0,0 +1,46 @@
+:og:description: While most members choose to pay online, cash and checks need to be recorded in Apiary manually by an officer or project manager. Treasurers also have the ability to apply waivers.
+
+Accepting Payments
+==================
+
+While most members choose to pay online, cash and checks need to be recorded in Apiary manually by an officer or project manager.
+Treasurers also have the ability to apply waivers.
+
+.. hint::
+   To record cash or check payments, you must have a project manager or officer role.
+   To apply waivers, you need a separate permission.
```

Comment:
We call them "roles" and "permissions" in the code, but I'm not sure those are good names for user-facing documentation. Open to suggestions here.

Reply:
I feel like it would be more clear at least to people in IT if we just list the exact role/permission names required in Apiary